bsc-plugin/src/lib/rooibos/MockUtil.ts

```diff
@@ -79,29 +77,53 @@ export class MockUtil {
             return;
         }
 
+        let isDisabledFoMocking = functionStatement.annotations?.find(x => x.name.toLowerCase() === 'disablemocking');
```

Comment:
is this a part of another pr? I'm happy with it; but it seems beyond the scope of this prs description.